### PR TITLE
add support for solaredge SE-MTR-3Y-* over RS485

### PR DESCRIPTION
**Pull request by meyerd**
Hi,
I just had the meter SE-MTR-3Y-400V installed, but no inverter shipped yet. Yet I was curious in my consumption prior to installing the inverter. If you are in the same situation and want to collect some data prior to buying an inverter, but yet want to re-use the meter afterwards, then this is for you.

Thanks for the project. Very helpful.

Cheers.

### CODE REVIEW

File: meters/rs485/semtr3y.go [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+		ImportL3:		0x0456, // imported active energy l3
+		Export:			0x0458, // total exported active energy	
+		ExportT1:		0x045A, // total exported active energy non-reset
+		ExportL1:		0x045C, // exported energy l1
+		ExportL2:		0x045E, // exported energy l2
+		ExportL3:		0x0460, // exported energy l3
+		ReactiveSum:		0x0462, // total reactive energy
+		ReactiveSumL1:		0x0464, // reactive energy l1
+		ReactiveSumL2:		0x0468, // reactive energy l2
+		ReactiveSumL3:		0x046A, // reactive energy l3
```
Comment (andig) [inline meters/rs485/semtr3y.go:66]: Plese use //

File: meters/rs485/semtr3y.go [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+func NewSEMTRProducer() Producer {
+	/**
+	 * Opcodes as defined by SolarEdge SE-MTR-3Y
+	 * reverse engineered from: https://github.com/nmakel/solaredge_meterproxy/blob/master/semp-rtu.py
+	 *
+	 * These are only necessary, if you'd like to connect the power meter
+	 * directly via rs485 (e.g. if you have no inverter from solaredge).
+	 * Otherwise, the values can be accessed readily through the network
+	 * connection of the inverter via the sunspec protocol.
+	 *
```
Comment (andig) [inline meters/rs485/semtr3y.go:30]: Remove empty line

File: meters/rs485/semtr3y.go [hunk 3]
```diff
@@ -0,0 +1,135 @@
+package rs485
+
+import (
+	"math"
+
+	. "github.com/volkszaehler/mbmd/meters"
+)
```
Comment (andig) [inline meters/rs485/semtr3y.go:7]: Remote meters import if not needed

**Review by andig — COMMENTED** [review]
Please. Re-generate docs, too so it appears in the readme (mbmd doc)

**Comment by andig** [comment]
Could you also add a note if the meter needs RTU or TCP encoding? I would assume RTU?

**Comment by andig** [comment]
ping @meyerd

**Comment by meyerd** [comment]
Hi @andig, 
sorry didn't notice the pull request was already worked on. Gmail filtered the github messages as spam. I'll have a look later today.

File: meters/rs485/semtr3y.go (hunk 3 above)
Comment (andig) [reply meters/rs485/semtr3y.go:6]: what  is the meters import for?

**Comment by meyerd** [comment]
@andig, actually, hold off with merging. there seems something fishy in the data and i guess the addresses for the second block of modbus readout values are off.

File: meters/rs485/semtr3y.go (hunk 3 above)
Comment (meyerd) [reply meters/rs485/semtr3y.go:6]: For the ImportL1, Sum etc. constants

**Comment by andig** [comment]
Formatting still seems wrong- are you using vscode or gofmt?

**Comment by meyerd** [comment]
> Formatting still seems wrong- are you using vscode or gofmt?

Neither, just vim. :) Sorry about the code quality. Never coded in GO before and don't know the specifics there ... 
Did a reformat with gofmt.

**Comment by andig** [comment]
I should've setup CI. Another day...

**Comment by andig** [comment]
Great PR, thanks a lot!

**Pull request merged**